src/pages/contact_us/_offices.tsx

```diff
@@ -182,7 +188,7 @@ const StyledMobileAddress = styled.div`
     }
 `
 
-export const Offices = () => {
+export const Offices: React.FC<GridLayoutProps> = () => {
```

Comment:
@NikitaK-deriv  no need to pass the `GridLayoutProps` here. It's already in the `GridLayout`  also we are moving away from `React.FC` you can just add type on the return as `ReactElement`

```
export const Offices = ():ReactElement => {
```